openmetadata-ui/src/main/resources/ui/src/components/GlossaryTerms/tabs/GlossaryOverviewTab.test.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { act, findByText, render, screen } from '@testing-library/react';
+import { MOCKED_GLOSSARY_TERMS, MOCK_PERMISSIONS } from 'mocks/Glossary.mock';
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import GlossaryOverviewTab from './GlossaryOverviewTab.component';
+
+jest.mock('components/GlossaryTerms/tabs/GlossaryTermSynonyms', () => {
+  return jest.fn().mockReturnValue(<p>GlossaryTermSynonyms</p>);
+});
+jest.mock('components/GlossaryTerms/tabs/RelatedTerms', () => {
+  return jest.fn().mockReturnValue(<p>RelatedTerms</p>);
+});
+jest.mock('components/GlossaryTerms/tabs/GlossaryTermReferences', () => {
+  return jest.fn().mockReturnValue(<p>GlossaryTermReferences</p>);
+});
+
+jest.mock('components/common/description/DescriptionV1', () => {
+  return jest.fn().mockReturnValue(<p>Description</p>);
+});
+
+describe('GlossaryOverviewTab', () => {
+  const onUpdate = jest.fn();
+  const selectedData = MOCKED_GLOSSARY_TERMS[0];
+  const permissions = MOCK_PERMISSIONS;
+  const isGlossary = true;
+
+  beforeEach(() => {
+    onUpdate.mockClear();
+  });
+
+  it('renders the component', async () => {
```

Comment:
- Consider passing empty value to reviewer, Synonyms, Tags, Terms and verify no data placeholders
- Add button if data isn't there
- Edit button if data is there

Reply:
These cases mentioned here are written in individual components. These are unit tests.